src/main/java/at/ac/tuwien/kr/alpha/solver/DefaultSolver.java

```diff
@@ -28,16 +28,10 @@
 package at.ac.tuwien.kr.alpha.solver;
 
 import at.ac.tuwien.kr.alpha.common.*;
-import at.ac.tuwien.kr.alpha.common.atoms.Atom;
-import at.ac.tuwien.kr.alpha.common.atoms.BasicAtom;
-import at.ac.tuwien.kr.alpha.common.atoms.ComparisonAtom;
-import at.ac.tuwien.kr.alpha.common.atoms.Literal;
+import at.ac.tuwien.kr.alpha.common.atoms.*;
```

Comment:
I suggest avoiding "star imports". These can seriously affect the code's robustness to future changes in case a type with a name that already exists in the namespace is added in one of the "star-imported" packages. This would then result in files that weren't even changed not compiling anymore. Detailed explanation [here](https://stackoverflow.com/questions/147454/why-using-a-wild-card-with-a-java-import-statement-bad)

Reply:
Fixed.